Apps/Contoso.Android.Puppet/ModulePages/AppCenterFragment.cs

```diff
@@ -130,5 +142,16 @@ private void WriteLog(object sender, EventArgs e)
             string tag = LogWriteTagText.Text;
             LogFunctions[mLogWriteLevel](tag, message);
         }
+
+        private void SaveStorageSize(object sender, EventArgs e)
+        {
+            var inputText = StorageSizeText.Text;
+            var size = string.IsNullOrEmpty(inputText) ? 0 : long.Parse(inputText);
```

Comment:
so, if inputText is not a number then we can easily get an exception here.
Not so safe approach. https://docs.microsoft.com/en-us/dotnet/api/system.int64.parse?view=net-5.0

Reply:
Hm. I thought we fixed all similar places based on my review some time ago - https://github.com/microsoft/appcenter-sdk-dotnet/pull/1471#discussion_r529422644
Good catch.
(JFYI it's part of test applications, not SDK, so I'd not block the release because of it)

Reply:
It will be addressed separately, I approved this PR